cmd/tether/msgs/messages.go

```diff
@@ -93,6 +99,28 @@ func (s *SignalMsg) Signum() int {
 	return Signals[s.Signal]
 }
 
+func (s *SignalMsg) FromString(name string) error {
+	num, err := strconv.Atoi(name)
+	if err == nil {
+		for sig, val := range Signals {
+			if num == val {
+				s.Signal = sig
+				return nil
+			}
+		}
```

Comment:
Will fallthrough to the error below: `return fmt.Errorf("unsupported signal name: %q", name)`